modules/metastorage-server/src/main/java/org/apache/ignite/internal/metastorage/server/SimpleInMemoryKeyValueStorage.java

```diff
@@ -353,7 +372,7 @@ private void compactForKey(
 
         Value lastVal = kv.get(key);
 
-        if (!lastVal.tombstone()) {
+        if (!lastVal.isTombstone()) {
```

Comment:
Please, return previous method name `tombstone` accordingly to https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-Gettersandsetters